configuration/defaults.go

```diff
@@ -118,9 +118,11 @@ vwIDAQAB
 	// defaultUserDeactivationFetchLimit the default number of accounts to return when fetching the users to deactivate (and deactivate at once)
 	defaultUserDeactivationFetchLimit = 10
 	// defaultUserDeactivationInactivityPeriod the default number of days of inactivity before a user account can be deactivated
-	defaultUserDeactivationInactivityNotificationPeriod = 24 // 24 days
+	//defaultUserDeactivationInactivityNotificationPeriod = 24 // 24 days
+	defaultUserDeactivationInactivityNotificationPeriod = 1 / 48 // 30 minutes
 	// defaultUserDeactivationInactivityPeriod the default number of days of inactivity before a user account can be deactivated
-	defaultUserDeactivationInactivityPeriod = defaultUserDeactivationInactivityNotificationPeriod + 7 // 31 days (7 days after defaultUserDeactivationInactivityNotificationPeriod)
+	//defaultUserDeactivationInactivityPeriod = defaultUserDeactivationInactivityNotificationPeriod + 7 // 31 days (7 days after defaultUserDeactivationInactivityNotificationPeriod)
+	defaultUserDeactivationInactivityPeriod = defaultUserDeactivationInactivityNotificationPeriod + 1/48 // 1 hour
```

Comment:
please check GetXXX() method for these variables.
defaultUserDeactivationInactivityNotificationPeriod [link](https://github.com/fabric8-services/fabric8-auth/blob/master/configuration/configuration.go#L1101)
defaultUserDeactivationInactivityNotificationPeriod [link](https://github.com/fabric8-services/fabric8-auth/blob/master/configuration/configuration.go#L1106)

The GetXXX() method multiply value of defaults with `default * 24 * time.Hour`  so I guess above default will not serve the purpose.